I.pass-tests/09-car-sales/car-sales.js

```diff
@@ -0,0 +1,31 @@
+function carSales(obj) {
+  let primeObj = obj.filter(num => {
+    return num.make == "Ford";
+  });
+
+  let secondObj = obj.filter(num => {
+    return num.make == "Honda";
+  });
+
+  let thirdObj = obj.filter(num => {
+    return num.make == "Land Rover";
+  });
+
+  let fourthObj = obj.filter(num => {
+    return num.make == "Toyota";
+  });
+  let primeP = primeObj[0].make;
+  let secondP = secondObj[0].make;
+  let thirdP = thirdObj[0].make;
+  let fuorthP = fourthObj[0].make;
+
+  let newObj = {
+    [primeP]: 22999,
```

Comment:
The total should be calculated. The way to generate this is to loop through the sold cards, add them to the map and add up the total sale price.